model-optimizer/extensions/back/NormalizeToNormalizeL2.py

```diff
@@ -28,6 +28,8 @@ class NormalizeToNormalizeL2(BackReplacementPattern):
     enabled = True
     force_clean_up = True
 
+    id = 'NormalizeToNormalizeL2_transform'
```

Comment:
can't we use a name of transformation itself? Or did you agree to insert such suffix `_transform` for all such transformations in case disabling it?

Reply:
Resolved using different approach.